docs/src/developing/clients/jsonrpc-api.md

```diff
@@ -1565,8 +1565,8 @@ Returns the account information for a list of Pubkeys
 - `<array>` - An array of Pubkeys to query, as base-58 encoded strings
 - `<object>` - (optional) Configuration object containing the following optional fields:
   - (optional) [Commitment](jsonrpc-api.md#configuring-state-commitment)
-  - `encoding: <string>` - encoding for Account data, either "base58" (*slow*), "base64", or jsonParsed". "base58" is limited to Account data of less than 128 bytes. "base64" will return base64 encoded data for Account data of any size.
-    Parsed-JSON encoding attempts to use program-specific state parsers to return more human-readable and explicit account state data. If parsed-JSON is requested but a parser cannot be found, the field falls back to base64 encoding, detectable when the `data` field is type `<string>`.
+  - `encoding: <string>` - encoding for Account data, either "base58" (*slow*), "base64", or "jsonParsed". "base58" is limited to Account data of less than 128 bytes. "base64" will return base64 encoded data for Account data of any size.
```

Comment:
Here's one.  There should be another on every changed line that starts with "encoding:" in this diff. I only counted five instead of six.  Also, searching [this file at this revision](https://github.com/solana-labs/solana/blob/d6c039decb1affbf914f5746143831a47473a527/docs/src/developing/clients/jsonrpc-api.md) for ' jsonParsed"' (with leading space) returns no results

Reply:
Sweet! I wonder if I was getting some weird cached display from github, because I see them now. Thanks!

Reply:
Weird caching? On Github? Certainly not! :wink: